package.json

```diff
@@ -5,11 +5,16 @@
   "author": "Karin Strouhalů <karin.strouhalu@nic.cz>",
```

Comment:
`name` should be `@cz-nic/django-forms-vuetify`, not `django-forms-vuetify`

Reply:
Fixed